collections/_infographics/emissions/emission-reduction-potential.md

```diff
@@ -3,12 +3,139 @@ layout:        infographic
+
+### Cutting emissions from landfills in half
+
+One of the areas where the Czech Republic's emissions have been increasing since 1990 is waste management. Approximately 2.7 million tonnes of waste end up in landfills yearly, the decomposition of which produces mainly methane, a potent greenhouse gas. The annual greenhouse gas emissions from landfills are equivalent to 3.7 million tonnes of <glossary id="co2eq">CO<sub>2</sub>eq</glossary>, which is 80% more than in 1990. If emissions from landfill could be halved, this would save 1.8 million tonnes of CO<sub>2</sub>eq per year.
+
+There are many ways to reduce the amount of waste going to landfill. It is possible to create products so they do not generate waste or follow the principles of the circular economy and use unused residues in one production process as raw material for further production. Further potential lies in the increased sorting of bio-waste and its targeted decomposition into biogas or the capture of methane from landfill bodies, which can then be used as fuel.
+
+### Planting 10 million extra trees
+
+Tree planting is seen as an important tool for carbon <glossary id="sequestration">sequestration</glossary>, which is reflected in some policy <!--[-->statements<!--](https://www.facebook.com/AndrejBabis/posts/1636697499800221/)-->. Ten million trees are chosen symbolically (one tree per citizen of the Czech Republic) and also in connection with the same goal of the project Planting the Future, founded by the Partnership Foundation and supported by the Ministry of the Environment. The project aims to plant 10 million new trees outside the forest area by 2025 (i.e. creating new alleys, orchards, thickets, windbreaks, etc.).
```

Comment:
Not sure how to handle this link to FB post: https://www.facebook.com/AndrejBabis/posts/1636697499800221/